src/pages/content-history.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
                                         m("a", {
                                                 title  : "Preview",
                                                 href   : m.route() + "/" + data.version,
-                                                config : m.route
+                                                target : "_blank"
```

Comment:
I just did it for consistency. Not particularly wed to it one way or the other. I can change it back at your say so.